x-pack/platform/plugins/private/observability_ai_assistant_management/public/routes/components/settings_tab/ui_settings.tsx

```diff
@@ -36,7 +35,6 @@ export function UISettings({ knowledgeBase }: { knowledgeBase: UseKnowledgeBaseR
     aiAnonymizationSettings,
     aiAssistantSimulatedFunctionCalling,
     ...(knowledgeBase.status.value?.enabled ? [aiAssistantSearchConnectorIndexPattern] : []),
-    ...(config.visibilityEnabled ? [aiAssistantPreferredAIAssistantType] : []),
```

Comment:
Since this has been removed, it can be removed from here:

x-pack/platform/plugins/shared/observability_ai_assistant/common/ui_settings/settings_keys.ts
x-pack/platform/plugins/shared/observability_ai_assistant/public/index.ts